lib/policy_machine_storage_adapters/active_record/postgresql.rb

```diff
@@ -74,6 +74,36 @@ def self.operations_for_operation_sets(operation_set_ids, operation_names = nil)
         # ]
         connection.execute(sanitized_query)
       end
+
+      def self.accessible_objects_for_operations(user_id, operation_names, options)
```

Comment:
can we toss in a comment like "this func assumes that the fields option will be passed with a single item"?

Reply:
d4b0485